faithcomesbyhearing/dbp
# FCBH-2584 update composer dependencies

# Description
- Updated composer dependencies
- Updated npm dependencies

## Issue Link
Original Story: [FCBH-2584](https://fullstacklabs.atlassian.net/browse/FCBH-2584) 

## How Do I QA This
- run `composer install`
- run `npm install`
- Test some endpoints and verify that are working correctly
